packages/components/src/components/common/table/tree/tree.tsx

```diff
@@ -21,7 +21,10 @@ const expanedButton = <TData,>(row: Row<TData>, lazy?: LazyExpand<TData>) => {
         className={expandButton}
         aria-label={row.getIsExpanded() ? 'Collapse row' : 'Expand row'}
         data-state={row.getIsExpanded() ? 'expanded' : 'collapsed'}
-        onClick={row.getToggleExpandedHandler()}
+        onMouseDown={row.getToggleExpandedHandler()}
+        onKeyDown={e => {
+          if (e.key === 'Enter' || e.key === ' ') row.getToggleExpandedHandler()();
+        }}
```

Comment:
removed the onclick, because this also triggers first onFocus, which selects the row (if it is in a SelectRow), but somehow not the click itself...
So, I changed it to an onMouseDown / onKeyDown event listener to toggle the expansion.